aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -2021,29 +2021,29 @@ Tensor mexp(const Tensor& a, bool compute_highest_degree_approx = false) {
 // Mathias, Roy.
 // A Chain Rule for Matrix Functions and Applications.
 // SIAM J. Matrix Anal. Appl. 17 (1996): 610-620.
-//
+
 template <typename func_t>
-Tensor backward_analytic_function_of_a_matrix(
+Tensor differential_analytic_matrix_function(
     const Tensor& self, const Tensor& grad,
-    const func_t& function_of_a_matrix
+    const func_t& matrix_function,
+    const bool backward_ad // Choose between forward or reverse AD
```

Comment:
"Choose between forward or reverse AD" implies that the variable would be better named as "reverse_ad"?

Reply:
Changed the comment :D